src/config/ConfigManager.cpp

```diff
@@ -1467,6 +1492,23 @@ void CConfigManager::parseLine(std::string& line) {
     parseKeyword(COMMAND, VALUE);
 }
 
+void CConfigManager::onOpenCategory() {
+    if (currentCategory.starts_with("submap")) {
+        const size_t index = currentCategory.find_first_of("-");
```

Comment:
const are CAPS

Reply:
Is there a way to do it with clang-format or something? It's VERY error prone.

Reply:
don't think so.